package.json

```diff
@@ -23,7 +23,14 @@
     "enforce": "istanbul check-coverage --statement 100 --branch 100 --function 100 --lines 100",
     "test": "NODE_ENV=test istanbul cover _mocha -- test --require test/setup.js --recursive --timeout 30000",
     "test:raw": "NODE_ENV=test mocha test --require test/setup.js --recursive --timeout 30000",
-    "start": "nodemon --ignore test/"
+    "start": "nodemon --ignore test/",
+    "db:migrate": "knex migrate:latest --knexfile db/index.js",
+    "db:migrate:make": "knex migrate:make --knexfile db/index.js",
+    "db:reset": "if [ \"$NODE_ENV\" != \"production\" ]; then dropdb $(node -p \"require('./config').DB_NAME\"); yarn run db:setup && yarn run db:migrate; fi",
+    "db:rollback": "knex migrate:rollback --knexfile db/index.js",
+    "db:setup": "if [ \"$NODE_ENV\" != \"production\" ]; then createdb -O $(node -e \"var c = require('./config'); console.log(c.DB_USER, c.DB_NAME);\"); fi",
+    "db:setup:user": "if [ \"$NODE_ENV\" != \"production\" ]; then createuser $(node -p \"require('./config').DB_USER\"); fi",
+    "db:seed": "yarn db:reset && knex seed:run --knexfile db/index.js"
```

Comment:
We typically like to alphabetize yarn scripts so that we can easily parse through if we're looking for a particular script -- could we move these `db` commands above `enforce`?

Reply:
and done!